src/sentry/snuba/spans_rpc.py

```diff
@@ -92,7 +92,7 @@ def run_table_query(
         limit=limit,
         virtual_column_contexts=[context for context in contexts if context is not None],
     )
-    rpc_response = snuba_rpc.table_rpc(rpc_request)
+    rpc_response = snuba_rpc.table_rpc([rpc_request])[0]
```

Comment:
Part of this is I'm changing to running multiple queries as the default way to interact with the snuba_rpc module, with typing i'm hoping this means its more intuitive that the capability exists